security/pkg/k8s/chiron/utils.go

```diff
@@ -94,11 +94,9 @@ func GenKeyCertK8sCA(client clientset.Interface, dnsName,
 // 2. Approve a CSR
 // 3. Read the signed certificate
 // 4. Clean up the artifacts (e.g., delete CSR)
-func SignCSRK8s(client clientset.Interface,
-	csrData []byte, signerName string, requestedDuration *time.Duration,
-	usages []certv1.KeyUsage,
-	dnsName, caFilePath string,
-	approveCsr bool, appendCaCert bool, requestedLifetime time.Duration) ([]byte, []byte, error) {
+func SignCSRK8s(client clientset.Interface, csrData []byte, signerName string, usages []certv1.KeyUsage,
+	dnsName, caFilePath string, approveCsr, appendCaCert bool, requestedLifetime time.Duration,
```

Comment:
Is this new formatting standard? I also got in couple of my changes. Wondering what triggered this?

Reply:
seems kind of weird to me as well but I guess so. I think it only happens on really long functions

I don't really care personally as long as it's automated and consistent

Reply:
Yeah I also do not care. Was googling and found these https://dave.cheney.net/2014/10/04/that-trailing-comma and https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8.

Was wondering what change is triggering these now- but as you said, as long as we are consistent it is good.